content/applications/sales/point_of_sale/restaurant.rst

```diff
@@ -1,15 +1,36 @@
-:nosearch:
+:show-content:
 
 ===================
-Restaurant Features
+Restaurant features
 ===================
 
+Managing a restaurant or a bar comes with specific needs. The Point of Sale application provides
+various features that allow performing all the required tasks in such businesses.
+
+Once the POS is set to be used in a restaurant or a bar, you can:
+
+- :doc:`Organize your floors and tables to reflect your interior, <restaurant/floors_tables>`
+- :ref:`Take orders <floors_tables/orders>`,
+- :doc:`Communicate with the kitchen or the bar through the POS, <restaurant/kitchen_printing>`
+- :doc:`Print bills in advance and split them, <restaurant/bill_printing>`
+- :doc:`Collect tips <restaurant/tips>`, and
+- :doc:`Set different taxes for takeaway food <restaurant/fiscal_position>`.
+
+Configuration
+=============
+
+To activate the features, go to :menuselection:`Point of Sale --> Configuration --> Point of Sale`,
+select your POS, and enable :guilabel:`Is a Bar/Restaurant`.
```

Comment:
```suggestion
select your POS, click :menuselection:`Edit --> Is a Bar/Restaurant --> Save`.
```
Since it's in 14, it requires a few more steps than just enabling a feature like in 16